tests/discovery_fixtures.py

```diff
@@ -22,6 +22,29 @@ class DiscoveryResponse(TypedDict):
     error_code: int
 
 
+UNSUPPORTED_HOMEWIFISYSTEM = {
+    "error_code": 0,
+    "result": {
+        "channel_2g": "10",
+        "channel_5g": "44",
+        "device_id": "REDACTED_51f72a752213a6c45203530",
+        "device_model": "X20",
+        "device_type": "HOMEWIFISYSTEM",
+        "factory_default": False,
+        "group_id": "12345623107d902da02fa9beab8a64",
+        "group_name": "TDBiaXQwV2lmaQ==",
+        "hardware_version": "3.0",
+        "ip": "192.168.1.192",
+        "mac": "24:2F:D0:00:00:00",
+        "master_device_id": "8228ddc1f51f72a752213a6c45203530",
```

Comment:
This should also probably be redacted just like `device_id` above.

Reply:
Good point, that's done along with group id and group name (which is the ssid)